src/InstrumentationEngine/ProfilerManager.h

```diff
@@ -252,12 +258,18 @@ namespace MicrosoftInstrumentationEngine
         // as that would screw up the com state for the application thread. This thread allows the profiler manager
         // to co create a free threaded version of msxml on a thread that it owns to avoid this.
         //
-        // lpParameter points to the this pointer so m_configFilePaths can be accessed. This is safe because the thread proc
+        // lpParameter points to the this pointer so m_configSources can be accessed. This is safe because the thread proc
         // blocks until this is complete.
         static DWORD WINAPI InstrumentationMethodThreadProc(
             _In_  LPVOID lpParameter
             );
 
+        // lpParameter points to the this pointer so m_wszConfigXml can be accessed. This is safe because the thread proc
+        // blocks until this is complete.
+        static DWORD WINAPI InitializeForAttachThreadProc(
```

Comment:
I think ParseAttachConfigurationThreadProc better describes what this is method is doing.